.github/workflows/basic.yml

```diff
@@ -44,7 +44,9 @@ jobs:
       - name: Setup Rust toolchain and cache
         uses: actions-rust-lang/setup-rust-toolchain@v1
         with:
-          components: rustfmt, clippy
+          components: clippy
+      - name: Install nightly rustfmt
+        run: rustup toolchain install nightly --component rustfmt --profile minimal --no-self-update
```

Comment:
nit: can we add the nightly directly in the setup rust step? If not can we have a comment here mentioning we use this for formatting?

Reply:
> can we add the nightly directly in the setup rust step?

That would result in using nightly `clippy`, which is not my intention here.
I'm open to suggestions :-)

Reply:
No, I can't think of anything better. We can keep this as is :+1: